redis-persistence/src/main/java/com/netflix/conductor/dyno/DynoProxyDiscoveryProvider.java

```diff
@@ -0,0 +1,33 @@
+package com.netflix.conductor.dyno;
+
+import com.netflix.conductor.core.config.Configuration;
+import com.netflix.discovery.DiscoveryClient;
+import com.netflix.dyno.jedis.DynoJedisClient;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+
+import redis.clients.jedis.JedisCommands;
+
+public class DynoProxyDiscoveryProvider implements Provider<JedisCommands> {
+    private final DiscoveryClient discoveryClient;
+    private final Configuration configuration;
+
+    @Inject
+    public DynoProxyDiscoveryProvider(DiscoveryClient discoveryClient, Configuration configuration) {
+        this.discoveryClient = discoveryClient;
+        this.configuration = configuration;
+    }
+
+    @Override
+    public JedisCommands get() {
+        String cluster = configuration.getProperty("workflow.dynomite.cluster", null);
```

Comment:
**Question**: Could we inject a more specific `Configuration`, so that `Configuration#getCluster()` can be used?